content/dst/codex/codex-comparison.md

```diff
@@ -5,14 +5,14 @@ tags:
   - "dst"
   - "codex"
 draft: false
-description: "Measure Codex against systems like IPFS, BitTorrent etc and see how it compares. Primarily BitTorrent."
+description: "Measure Codex against BitTorrent and see how it compares."
 ---
 
 `vac:dst:codex:codex-comparison`
-Measure Codex against systems like IPFS, BitTorrent etc and see how it compares. Primarily BitTorrent.
+Measure Codex against BitTorrent.
 ## Description
-We will compare Codex to other systems like IPFS and BitTorrent 
-to see how it performs.
+We will compare Codex to other systems - for this initial phase, BitTorrent -
+to see how it performs in comparison.
```

Comment:
It is not just that one - we have all these

```
### How Fast Is Codex?
* fully qualified name: <vac:dst:codex:codex-scaling:how-fast-is-codex>
```
and

```
### Working Matrices

* fully qualified name: <vac:dst:ift:deployer-tool:working-matrices>
```

and

```
### ArgoCD Or Similar

* fully qualified name: `vac:dst:ift:deployer-tool:argocd-or-similar`
```

we have about 3 tasks, they have maybe 4, and in terms of work it's actually pretty much even :pray:

Reply:
I will leave this comment unresolved for now, and a TODO for a future PR is to make this split clearer by moving the tasks into one commitment that captures it all. This would involve merging 

* fully qualified name: <vac:dst:ift:deployer-tool:working-matrices>
* fully qualified name: `vac:dst:ift:deployer-tool:argocd-or-similar`

into Codex in one commitment.

Reply:
https://github.com/vacp2p/roadmap/pull/22 maybe addressed here